targets/TARGET_STM/TARGET_STM32G0/serial_device.c

```diff
@@ -93,52 +88,35 @@ static void uart1_irq(void)
 }
 #endif
 
-#if defined(USART2_BASE)
-static void uart2_irq(void)
+#if defined (USART_GROUP1_IRQn)
+static void uart_group1_irq(void)
 {
+#if defined(USART2_BASE)
```

Comment:
This line has to be removed ?

Reply:
Actually it appeared to be missing an #endif, fixed in fe698ea13b5bf97477fbbe04fbfa081015207be2